src/MemorySquare/MemorySquare.js

```diff
@@ -3,11 +3,9 @@ import { Link } from 'react-router-dom'
 
 class MemorySquare extends React.Component {
   render () {
-    const pathname = '/memory/' + this.props.memory._id
     return (
       <div className='MemorySquareContainer'>
-        <img src={this.props.memory.imageUrl} />
-        <Link to={pathname}>Testing</Link>
+        <Link to={`/memory/${this.props.memory._id}`}><img src={this.props.memory.imageURL} /></Link>
```

Comment:
have title also render in a h3 or h4 element. I can style the element over the image in white text